src/components/Profile/MyPosts/Post/Post.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+
+import s from './Post.module.css';
```

Comment:
Consider renaming "s" to "styles" to be more descriptive.

Reply:
/fxd